auth/src/main/java/com/redhat/lightblue/rest/auth/ldap/LightblueLdapRoleProvider.java

```diff
@@ -43,10 +40,10 @@
     InitialLdapContextProvider ldapContextProvider;
     String ldapSearchBase;
 
-    public LightblueLdapRoleProvider(String server, String searchBase, String bindDn, String bindDNPwd) throws NamingException {
+    public LightblueLdapRoleProvider(String server, String searchBase, String bindDn, String bindDNPwd, Boolean useSSL, String trustStore, String trustStorePassword) throws NamingException {
```

Comment:
Perhaps, let me add the default one back, and revert the LightblueLdapRoleProviderTest change, and we can revisit this when we get to changing out the LDAP library (see https://github.com/lightblue-platform/lightblue-rest/issues/257)